src/control/conf.h

```diff
@@ -215,7 +215,7 @@ static inline void dt_conf_init(dt_conf_t *cf, const char *filename, GSList *ove
     if(!i)
     {
       snprintf(darktable.conf->filename, sizeof(darktable.conf->filename), "%s", filename);
-      f = fopen(filename, "rb");
+      f = fopen(filename, "r");
```

Comment:
You can drop the changes to this function, I changed the way the files are read and tested it on Linux and Windows with both kinds of line endings. Please check and verify that what is currently in master works for you.